packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdk.kt

```diff
@@ -219,6 +221,37 @@ class DdSdk(
             configBuilder.setProxy(proxy, authenticator)
         }
 
+        _InternalProxy.setTelemetryConfigurationEventMapper(
+            configBuilder,
+            object : EventMapper<TelemetryConfigurationEvent> {
+                override fun map(event: TelemetryConfigurationEvent): TelemetryConfigurationEvent? {
+                    event.telemetry.configuration.trackNativeErrors =
+                        configuration.nativeCrashReportEnabled
+                    // trackCrossPlatformLongTasks will be deprecated for trackLongTask
+                    event.telemetry.configuration.trackCrossPlatformLongTasks =
+                        configuration.longTaskThresholdMs != 0.0
+                    event.telemetry.configuration.trackLongTask =
+                        configuration.longTaskThresholdMs != 0.0
+                    event.telemetry.configuration.trackNativeLongTasks =
+                        configuration.nativeLongTaskThresholdMs != 0.0
+
+
+                    event.telemetry.configuration.initializationType =
+                        configuration.configurationForTelemetry?.initializationType
+                    event.telemetry.configuration.trackInteractions =
+                        configuration.configurationForTelemetry?.trackInteractions
+                    event.telemetry.configuration.trackErrors =
+                        configuration.configurationForTelemetry?.trackErrors
+                    event.telemetry.configuration.trackResources =
+                        configuration.configurationForTelemetry?.trackNetworkRequests
+                    event.telemetry.configuration.trackNetworkRequests =
```

Comment:
what is the difference between the two?

Reply:
According to https://github.com/DataDog/rum-events-format/blob/master/schemas/telemetry/configuration-schema.json:

- track_network_requests : Whether automatic collection of network requests is enabled
- track_resources: Whether resources are tracked

Also `track_network_requests` is supposed to apply to mobile only, I can't trace back why we made the decision to go with a new parameter instead of using `track_resources` that already existed for browser.

My thoughts were that in the React Native SDK, both are always on or off together (there are no other resources tracked other than network requests, and we can only collect them automatically).
Actually, `trackResources` does not seem to be used by the Android nor iOS SDK, so I can drop it.